app/uk/gov/hmrc/eoricommoncomponent/frontend/views/migration/how_can_we_identify_you.scala.html

```diff
@@ -17,16 +17,16 @@
 @import uk.gov.hmrc.eoricommoncomponent.frontend.domain.NinoOrUtr
 @import uk.gov.hmrc.eoricommoncomponent.frontend.views.html._
 @import uk.gov.hmrc.eoricommoncomponent.frontend.models.Service
+@import views.html.helper._
+@import uk.gov.hmrc.eoricommoncomponent.frontend.controllers.registration.routes._
 
 
 @this(layout_di: layout)
 @(ninoUtrForm: Form[NinoOrUtr], isInReviewMode: Boolean, service: Service, journey: Journey.Value, organisationType: Option[String] = None)(implicit request: Request[_], messages: Messages)
 
-@import views.html.helper._
-@import uk.gov.hmrc.eoricommoncomponent.frontend.controllers.registration.routes._
-@inputText(field: String, label: String, args: (Symbol, Any)*) = {
+@inputText(field: String, label: String) = {
 @defining(Seq(
-    '_label -> Html(messages(label))) ++ args) { args => @helpers.input(ninoUtrForm(field), args: _*) }
+    '_label -> Html(messages(label)), '_plainLabel -> true)) { args => @helpers.input(ninoUtrForm(field), args: _*) }
```

Comment:
I can understand what this Defining Seq is doing, but are we not able to somehow fix this from component perspective? Those settings looks little "hacky".

but I see that we're in the same way defining the label, so it's okay for me.

Reply:
The `input` component is sometime used to render bold labels (where they should be bold).  So this is apparently the way the original author passes in params to the component.

The sooner we re-write the views to use GDS components the better.

Reply:
Yes, agree